aiven_mysql_migrate/migration.py

```diff
@@ -348,7 +348,7 @@ def _start_replication(self):
         with self.target_master.cur() as cur:
             query = (
                 "CHANGE MASTER TO MASTER_HOST = %s, MASTER_PORT = %s, MASTER_USER = %s, MASTER_PASSWORD = %s, "
-                "MASTER_AUTO_POSITION = 1, MASTER_SSL = 1"
+                f"MASTER_AUTO_POSITION = 1, MASTER_SSL = {int(self.source.ssl)}"
```

Comment:
I think if would be clearer/safer to say `1 if self.source.ssl else 0`: `MySQLConnectionInfo.ssl` is marked as `Optional`, the conversion to int will fail if the ssl value is None.

( However, in the entire repo, `MySQLConnectionInfo` objects are only created by `from_uri`, which never sets `ssl` to `None`, maybe we should also modify that `ssl` declaration and remove that `Optional`. )

Reply:
True value will be casted to 1, any other value to 0, that's what I want.

Reply:
True value will be converted to 1, False value will be converted to 0, but None value will raise a TypeError instead of returning 0:
```
>>> int(None)
Traceback (most recent call last):
  File "<input>", line 1, in <module>
TypeError: int() argument must be a string, a bytes-like object or a number, not 'NoneType'
```

Reply:
oops, true :) alright